python/lsst/resources/gs.py

```diff
@@ -197,12 +193,13 @@ def mkdir(self) -> None:
         self.blob.upload_from_string(b"", retry=_RETRY_POLICY)
 
     def _as_local(self) -> tuple[str, bool]:
-        with tempfile.NamedTemporaryFile(suffix=self.getExtension(), delete=False) as tmpFile:
-            with time_this(log, msg="Downloading %s to local file", args=(self,)):
-                try:
-                    self.blob.download_to_filename(tmpFile.name, retry=_RETRY_POLICY)
-                except NotFound as e:
-                    raise FileNotFoundError(f"No such resource: {self}") from e
+        with tempfile.NamedTemporaryFile(suffix=self.getExtension(), delete=False) as tmpFile, time_this(
+            log, msg="Downloading %s to local file", args=(self,)
+        ):
```

Comment:
I'm not sure that this is easier to read than original.

Reply:
I wanted to write it as:

```python
with (
    tempfile.NamedTemporaryFile(...) as tmpFile,
    time_this(...)
):
    ...
```

but black rewrites it...

Reply:
OK, we are all robots at this point, anyways. 🤖 🙂

Reply:
I am guessing that black hasn't been taught about parenthesized context managers yet.